lib/fastlane/plugin/wpmreleasetoolkit/helper/ci_helper.rb

```diff
@@ -0,0 +1,58 @@
+require 'fastlane_core/ui/ui'
+require 'net/http'
+require 'uri'
+require 'json'
+
+module CIHelper
+    def trig_job(branch, parameters=nil)
+      raise "Not implemented"
+    end
+
+    def login
+        raise "Not implemented"
+    end
+
+    def organization
+        raise "Not implemented"
+    end
+
+    def repository 
+        raise "Not implemented"
+    end
+end
+
+module Fastlane
+    module Helper
+        class CircleCIHelper 
+            include CIHelper
+
+            attr_accessor :login, :organization, :repository
+
+            def initialize(login, repository, organization = "wordpress-mobile")
```

Comment:
Could be nice to make use of named parameters here too
```suggestion
            def initialize(login:, repository:, organization: "wordpress-mobile")
```